src/stdpar/CondFormats/PixelCPEFast.cc

```diff
@@ -14,57 +13,31 @@
 //-----------------------------------------------------------------------------
 //!  The constructor.
 //-----------------------------------------------------------------------------
-PixelCPEFast::PixelCPEFast(std::string const &path) {
+PixelCPEFast::PixelCPEFast(std::string const &path)
+    : m_params{std::make_shared<pixelCPEforGPU::ParamsOnGPU>()},
+      m_commonParams{std::make_shared<pixelCPEforGPU::CommonParams>()},
+      m_layerGeometry{std::make_shared<pixelCPEforGPU::LayerGeometry>()},
+      m_averageGeometry{std::make_shared<pixelCPEforGPU::AverageGeometry>()} {
```

Comment:
Why `shared_ptr` and not `unique_ptr`?

Reply:
I actually had a local commit which use `unique_ptr`. I was using a `shared_ptr` because data members were also taking a pointer to these structures.